Testscripts/Linux/build_install_rdma_core.sh

```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache License.
+
+# This script will build and install rdma-core
+
+HOMEDIR=$(pwd)
+UTIL_FILE="./utils.sh"
+
+# Source utils.sh
+. utils.sh || {
+	echo "ERROR: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 0
+}
+
+# Source constants file and initialize most common variables
+UtilsInit
+
+function build_install_rdma_core () {
+	SetTestStateRunning
+	packages=(git)
+	case "${DISTRO_NAME}" in
```

Comment:
https://github.com/linux-rdma/rdma-core provides the each distro's installation steps.

Reply:
Currently, the rdma-core install/build is a hack to make the primary/secondary DPDK processes work on Azure, as the default version of the rdma-core package is lower than 23.

Once the rdma-core patches are backported or the version is changed in the supported distros, this part will be removed.

Note that this test case is supposed to work only with Linux kernel 4.20 or higher, DPDK 19.05 or higher, rdma-core 23 or higher.

The only Azure image that is close to these specifications is Ubuntu 19.04 (which has to have rdma-core upgraded, as the current default version is version 23).

Not even RHEL 8 has kernel > 4.20, it has 4.18.